app.rb

```diff
@@ -0,0 +1,73 @@
+require_relative 'lib/person/student'
+require_relative 'lib/person/teacher'
+require_relative 'lib/book'
+require_relative 'lib/rental'
+
+class App
+  attr_reader :books, :people, :rentals
+
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_books
+    if @books.empty?
+      print 'There are no registered books'
+    else
+      print 'Books:'
+      @books.each do |book|
+        print "Title: #{book.title}, Author: #{book.author}"
+      end
+    end
+  end
```

Comment:
- [ ] You did an excellent job, but every time I attempted to list all the books, the application would unexpectedly shut down. Kindly address this issue :+1: 

![Screenshot from 2023-10-20 17-55-49](https://github.com/BiancaMe/school-library/assets/105117832/3f0d8635-c62c-4dc2-937f-3e4828b9a35f)